ui/core/src/model/project.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Metadata } from '@perses-dev/core';
```

Comment:
Does this make webpack think there is a circular dependency? I’d usually expect this to be a relative path so that core package isn’t importing from its own barrel file (I thought we had an eslint rule for this but maybe not)

Reply:
Yep, I think it should fail the build, I already saw this rule on other package. Issue fixed, but I didn't check the rule for now